src/arch/riscv/objects.mk

```diff
@@ -15,3 +15,6 @@ cpu-objs-y+=cpu.o
 cpu-objs-y+=cache.o
 cpu-objs-y+=iommu.o
 cpu-objs-y+=relocate.o
+ifeq ($(IPIC),IPIC_ACLINT)
+cpu-objs-y+=aclint.o
+endif
```

Comment:
I understand that we have this so that we can use both configurations for qemu. However, this is not really that useful for other "real" platforms that either have an SSWI or not. I don't like the idea of "polluting" the arch makefiles with these option. Maybe IPIC should only be present on  platform.mk.